kyaml/filesys/fsondisk_test_unix.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2022 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build aix || darwin || dragonfly || freebsd || (js && wasm) || linux || netbsd || openbsd || solaris
```

Comment:
```suggestion
//go:build !windows
```

Reply:
Will do. I just wanted to be careful at first because there seem to be some OSs that are not covered in this list and not windows.

Reply:
We only test on Linux, Mac, and Windows, so it doesn't make sense to include build tags in test files for other OS's; the only distinction we really need is windows vs not windows. IMO keeping the other OS tags is overcomplicating it unnecessarily and may instead be a point of confusion for another contributor who look at this and wonder why we have this long list of OSs here that we don't even support.